schemas/telemetry/debug-schema.json

```diff
@@ -9,19 +9,23 @@
       "$ref": "_common-schema.json"
     },
     {
-      "required": [
-        "status",
-        "message"
-      ],
+      "required": ["telemetry"],
       "properties": {
-        "status": {
-          "type": "string",
-          "description": "Level/severity of the log",
-          "const": "debug"
-        },
-        "message": {
-          "type": "string",
-          "description": "Body of the log"
+        "telemetry": {
+          "type": "object",
+          "description": "The telemetry information as it should be indexed",
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** I feel that "as it should be indexed" is unnecessary, since other fields should be indexed as well